### DISCUSSION TITLE
这个测试性能消耗大吗?mt7620只能跑2m的速度

### DISCUSSION THREAD

**Discussion by lijiazhiren** [XIU2/CloudflareSpeedTest#536] [提问]
最新版测试了好几次 
测试命令是 ./CloudflareST  不加其他参数
mt7620都精准的卡在了2m
pc测试则不是

**Comment by XIU2** [comment]
按理说，CloudflareST 对 处理器、内存、存储 都没啥需求的。

软件内也没啥需要大量计算的数据，我这边不管是电脑还是路由器上使用，CPU 占用都很低。
至于内存和存储就更没啥了，因为下载测速的话，接收到的数据是直接丢弃的，压根不需要写入内存和存储，软件只需要计算单位时间内接收了多少数据（然后除以时间=平均速度），因此下载测速过程中软件的内存占用是基本不变的，存储读写更是约等于没有（只有指定 IP 数据文件时，会读取；最后写入测速结果时，会写入）。

但确实有人反馈过在一些设备上下载测速慢的问题（要么是路由器，要么是手机，倒是没人反馈过电脑上的），这我也没有头绪。。。
而我家路由器 MT7981 平时使用是完全没有这个问题的。

所以目前这个问题，只能归属在 `疑难杂症` 里了。。。

**Reply by lijiazhiren** [reply]
一开始我在安卓手机测试过那个推荐的app
速度好像也没有超过2兆 而且出现了很多error
以为是app有问题 
才又折腾了一下这个mt7620的openwrt路由 没想到精准的卡在2m
然后我在pc的虚拟机上测试了一下x86的openwrt
随便一测就直接测出跑满带宽的ip
可能真是疑难杂症了
